lib/reporter/logger.js

```diff
@@ -1,24 +1,31 @@
 'use strict';
-const _ = require('lodash');
 const chalk = require('chalk');
 
 const util = require('./util');
 
+const outChunkSize = 10000;
+
 function logModule(log, logType, module) {
   log[logType](chalk.yellow('module name:'), module.name);
   if (!module.skipped) {
     log[logType](chalk.yellow('version:'), module.version);
   }
   if (module.error) {
     log[logType]('error:', module.error.message);
-    log[logType]('error:', module.testOutput);
+    // If the output is too big the log formatter explodes.
+    for (let i = 0; i < module.testOutput.length; ) {
+      const rawChunk = module.testOutput.substring(i, i += outChunkSize);
+      // Align chunk to EOL.
+      const lastEOL = rawChunk.lastIndexOf('\n') + 1;
+      i -= (outChunkSize - lastEOL);
+      const chunk = rawChunk.substring(0, lastEOL);
+      log[logType]('error:', chunk);
+    }
```

Comment:
Would you be so kind and add a more detailed explanation? The current code is somewhat confusing to me.

Reply:
Ack